package.json

```diff
@@ -10,10 +10,10 @@
     "build:js": "vite build",
     "test:unit": "vitest run",
     "test:e2e": "test:e2e:all",
-    "test:e2e:all": "npm-run-all -p start test:cypress:all",
-    "test:e2e:one": "npm-run-all -p start test:cypress:one",
-    "test:cypress:all": "cypress run",
-    "test:cypress:one": "cypress open",
+    "test:e2e:all": "VITE_TRACTION_BASE_URL=http://localhost:3004 npm-run-all -p start test:cypress:all",
```

Comment:
Line 13 and 14 are aliases for 15 and 16 so the prefix should only be needed on 15 & 16.

Reply:
My mistake - 'test:e2e:all' and 'test:e2e:one' actually call/alias to two commands so you need the override prefix there because otherwise the 'start' command wont pick up the override.

Reply:
I suspect this is why it appeared as though cypress wasn't picking up the env variables because 'start' doesn't pick them up